javascript/javascript3/week1/Giphy/index.js

```diff
@@ -0,0 +1,52 @@
+const userInput = document.querySelector(".gifName");
+let gifArray;
+document.querySelector(".search").addEventListener("click", getGif);
+async function getGif() {
+  if (userInput.value.trim() === "") {
```

Comment:
the best practice is to save the API-Url as a variable instead of repeating it every time. you need to fetch the data. 
for example 
```
`let limit = 10;
let searchTerm = "";
let apiUrl = `https://api.giphy.com/v1/gifs/search?api_key=Y55ch22OYmhAWV15dqMxf0PDOtgBzrlL&q=``
```
and then when you want to fetch data . you get searchTerm like this 
`fetch(apiUrl + userInput.value + &limit=${limit}&offset=0&rating=g&lang=en`);`